File: src/main/java/conditionalStatements/PickQuadrant.java
package conditionalStatements;

import java.util.Scanner;

public class PickQuadrant {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int number1 = scanner.nextInt();
        int number2 = scanner.nextInt();

        if (number1 > 0 && number2 > 0) {
            System.out.println("1");
        } else if (number1 < 0 && number2 > 0) {
            System.out.println("2");
        } else if (number1 < 0 && number2 < 0) {
            System.out.println("3");
        } else {
            System.out.println("4");
        }
    }

}

File: src/main/kotlin/conditionalStatements/PickQuadrant.kt
package conditionalStatements

import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)
    val x: Int = scanner.nextInt()
    val y: Int = scanner.nextInt()
    scanner.close()

    when {
        x > 0 && y > 0 -> println("1")
        x < 0 && y > 0 -> println("2")
        x < 0 && y < 0 -> println("3")
        x > 0 && y < 0 -> println("4")
    }

}